src/box/lua/config/instance_config.lua

```diff
@@ -1179,6 +1183,57 @@ return schema.new('instance_config', schema.record({
             end
         end,
     }),
+    feedback = schema.record({
+        crashinfo = schema.scalar({
+            type = 'boolean',
+            box_cfg = 'feedback_crashinfo',
+            default = true,
+            apply_default_if = feedback_apply_default_if,
+        }),
+        host = schema.scalar({
+            type = 'string',
+            box_cfg = 'feedback_host',
+            default = 'https://feedback.tarantool.io',
+            apply_default_if = feedback_apply_default_if,
+        }),
+        metrics_collect_interval = schema.scalar({
+            type = 'number',
+            box_cfg = 'feedback_metrics_collect_interval',
+            default = 60,
+            apply_default_if = feedback_apply_default_if,
+        }),
+        send_metrics = schema.scalar({
+            type = 'boolean',
+            box_cfg = 'feedback_send_metrics',
+            default = true,
+            apply_default_if = feedback_apply_default_if,
+        }),
+        enabled = schema.scalar({
+            type = 'boolean',
+            box_cfg = 'feedback_enabled',
+            default = true,
+            apply_default_if = feedback_apply_default_if,
+        }),
+        interval = schema.scalar({
+            type = 'number',
+            box_cfg = 'feedback_interval',
+            default = 3600,
+            apply_default_if = feedback_apply_default_if,
+        }),
+        metrics_limit = schema.scalar({
+            type = 'number',
+            box_cfg = 'feedback_metrics_limit',
+            default = 1024 * 1024,
+            apply_default_if = feedback_apply_default_if,
+        }),
+    }, {
+        validate = function(data, w)
+            if data == nil or box.internal.feedback_daemon ~= nil then
+                return
+            end
+            w.error('Feedback is disabled')
```

Comment:
I would make it clear that the feedback sending is disabled at compile time, not just using `feedback.enabled`.

For example,

```lua
w.error('Tarantool is built without feedback reports sending support')
```

Reply:
Done